New log file opened at Sun May 10 14:36:30 CEST 2015 by 10960@Julia
Taking next connection to: 57121
New client attempting to connect on port: 57122-57121
  Address = /10.0.0.3:57121
This client on port 57122 has uuid 0637fb3f-4def-4860-a270-5f330386646f configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-0637fb3f-4def-4860-a270-5f330386646f
Taking next connection to: 57121
New client attempting to connect on port: 57123-57121
  Address = /10.0.0.3:57121
Handshake exception on socket: 57123
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 57121
New client attempting to connect on port: 57124-57121
  Address = /10.0.0.3:57121
This client on port 57124 has uuid ab9231df-3cf3-4f98-a1c4-69276e6279df configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-0637fb3f-4def-4860-a270-5f330386646f, activator-ab9231df-3cf3-4f98-a1c4-69276e6279df
Taking next connection to: 57121
New client attempting to connect on port: 57129-57121
  Address = /10.0.0.3:57121
Handshake exception on socket: 57129
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 57121
New client attempting to connect on port: 57130-57121
  Address = /10.0.0.3:57121
This client on port 57130 has uuid c6a21658-9915-45b7-860f-611ade2891df configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-ab9231df-3cf3-4f98-a1c4-69276e6279df, play-fork-run-c6a21658-9915-45b7-860f-611ade2891df
Taking next connection to: 57121
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57121
